apps/todo-app/package.json

```diff
@@ -30,7 +30,7 @@
     "react": "16.8.6",
     "react-app-polyfill": "~1.0.1",
     "react-dom": "16.8.6",
-    "typescript": "3.7.5",
+    "typescript": "~3.7.2",
```

Comment:
Lock this too

Reply:
I tried pinning to `3.7.5` but API extractor is using `~3.7.2`. I've pinned it to `3.7.2` for now.